ui/pages/AnvilWorkspace/components/LoadWorkspaceDataForm.jsx

```diff
@@ -118,6 +134,7 @@ const LoadWorkspaceDataForm = React.memo(({ namespace, name }) => (
         see &nbsp;
         <b><a href={VCF_DOCUMENTATION_URL} target="_blank" rel="noreferrer">this documentation</a></b>
       </Message>
+      {WARNING_BANNER ? <Message error compact header={WARNING_HEADER} content={WARNING_BANNER} /> : null}
```

Comment:
You may want to add a date condition (before Jan 3, 2022) to show the warning because the seqr might not be updated right after the holidays.

Reply:
no I want this to show until whenever we explicitly decide to turn it off in January, we don't yet know what the end date should be